node/package.json

```diff
@@ -32,7 +32,7 @@
     "slugify": "^1.2.6",
     "typescript": "3.9.7",
     "unescape": "^1.0.1",
-    "vtex.search-graphql": "http://vtex.vtexassets.com/_v/public/typings/v1/vtex.search-graphql@0.39.0/public"
+    "vtex.search-graphql": "http://facetest--btglobal.vtexassets.com/_v/private/typings/linked/v1/vtex.search-graphql@0.39.0/public"
```

Comment:
can you undo this change?

Reply:
I will make this change as soon as I publish `search-graphql` in order to have the version and change it here.

Reply:
you can go to search-graphql, link your workspace there and test with the generated URL, so it doesn't need to change here